### ISSUE TITLE
controller flashes when it vibrates

### ISSUE THREAD

**Issue by vulturecarcass** [Jays2Kings/DS4Windows#491]

**Comment by vulturecarcass** [comment]
yeah i dont know what im doing but id like my controller to stop flashing when it vibrates how do i stop this lmao

**Comment by GingerAdonis** [comment]
Just as a heads up: This fork/version of DS4Windows is outdated. An improved version is being actively maintained by Ryochan7.

> https://github.com/Ryochan7/DS4Windows/releases
